const getSuccess = async (req, res) => {
    try {
        res.send("Success !");
    } catch (error) { }
};
export default getSuccess;
